cmake/tribits/examples/TribitsExampleProject/README

```diff
@@ -2,50 +2,36 @@
 # TribitsExampleProject Documentation
 #
 
-WARNING: WORK IN PROGRESS!  This TriBITS example project will get fleshed out
-more in the future to be more complete.
-
 The project TribitsExampleProject defines a TriBITS CMake project designed to
 provide a simple example to demonstrate how to use the TriBITS system to
-create a build/test system for any given project.
+create a CMake build, test, and deployment system using a package-based
+architecture.
 
 To get started building you can create a directory structure like:
 
     ~/TribitsExampleProject.base/
         TribitsExampleProject/   # This directory
         BUILDS/
             GCC-<X.Y.Z>/
-                MPI_DEBUG/
+                SERIAL_DEBUG/
 
 You can then copy the configure file as:
 
-  cd ~/TribitsExampleProject.base/BUILDS/GCC-<X.Y.Z>/BUILDS/MPI_DEBUG
-  cp ../../../../../../TribitsExampleProject/sampleScripts/linux-mpi-debug.sh do-configure
-
-You can create another directory for SERIAL_RELEASE or whatever build type you
-want.
+  cd ~/TribitsExampleProject.base/BUILDS/GCC-<X.Y.Z>/BUILDS/SERIAL_DEBUG
+  cp ../../../../TribitsExampleProject/sampleScripts/linux-serial-debug.sh \
+    do-configure
 
-If your system is already to set up to find the MPI compiler wrappers, you con
-just configure with:
+After that set, configure with:
```

Comment:
> After that set, configure with:

is confusing to me.  "After that is set, configure with:" or "After that, configure with:"